calamus/fields.py

```diff
@@ -335,7 +341,10 @@ def _dereference_flattened(self, value, attr, **kwargs):
 
     def _deserialize(self, value, attr, data, **kwargs):
         """Deserialize nested object."""
-        if "flattened" in kwargs and kwargs["flattened"]:
+
+        kwargs["_ignore_lazy"] = False
```

Comment:
What is the use of this?

Reply:
oh this was left over from a previous try, I'll remove it